OpenSim/Analyses/Test/testOutputReporter.cpp

```diff
@@ -0,0 +1,300 @@
+    //==========================================================================
+
+    // Define the initial and final simulation times
+    double initialTime = 0.0;
+    double finalTime = 0.5;
+
+    //Physical properties of the model
+    double ballMass = 10;
+    double ballRadius = 0.05;
+    double anchorWidth = 0.1;
+
+    // Create an OpenSim model
+    Model model;
+
+    double optimalFiberLength = muscModel.getOptimalFiberLength();
+    double pennationAngle     = muscModel.getPennationAngleAtOptimalFiberLength();
+    double tendonSlackLength  = muscModel.getTendonSlackLength();
+
+    // Use a copy of the muscle model passed in to add path points later
+    Muscle *muscle = muscModel.clone();
+
+    // Get a reference to the model's ground body
+    Ground& ground = model.updGround();
+
+    OpenSim::Body * ball = new OpenSim::Body("ball",
+        ballMass,
+        Vec3(0),
+                        ballMass*SimTK::Inertia::sphere(ballRadius));
+
+    ball->attachGeometry(new Sphere(ballRadius));
+    // ball connected  to ground via a slider along X
+    double xSinG = optimalFiberLength*cos(pennationAngle) + tendonSlackLength;
+
+    SliderJoint* slider = new SliderJoint("slider",
+        ground,
+        Vec3(anchorWidth / 2 + xSinG, 0, 0),
+        Vec3(0),
+        *ball,
+        Vec3(0),
+                        Vec3(0));
+
+    auto& sliderCoord = slider->updCoordinate();
+    sliderCoord.setName("tx");
+    sliderCoord.setDefaultValue(1.0);
+    sliderCoord.setRangeMin(0);
+    sliderCoord.setRangeMax(1.0);
+
+    // add ball to model
+    model.addBody(ball);
+    model.addJoint(slider);
+
+    //==========================================================================
+    // 1. SIMULATION SETUP: Add the muscle
+    //==========================================================================
+    //Attach the muscle
+    /*const string &actuatorType = */muscle->getConcreteClassName();
+    muscle->setName("muscle");
+    muscle->addNewPathPoint("muscle-box", ground, Vec3(anchorWidth / 2, 0, 0));
+    muscle->addNewPathPoint("muscle-ball", *ball, Vec3(-ballRadius, 0, 0));
+
+    ActivationFiberLengthMuscle_Deprecated *aflMuscle
+        = dynamic_cast<ActivationFiberLengthMuscle_Deprecated *>(muscle);
+    if (aflMuscle){
+        // Define the default states for the muscle that has 
+        //activation and fiber-length states
+        aflMuscle->setDefaultActivation(act0);
+        aflMuscle->setDefaultFiberLength(aflMuscle->getOptimalFiberLength());
+    }
+    else{
+        ActivationFiberLengthMuscle *aflMuscle2
+            = dynamic_cast<ActivationFiberLengthMuscle *>(muscle);
+        if (aflMuscle2){
+            // Define the default states for the muscle 
+            //that has activation and fiber-length states
+            aflMuscle2->setDefaultActivation(act0);
+            aflMuscle2->setDefaultFiberLength(aflMuscle2
+                ->getOptimalFiberLength());
+        }
+    }
```

Comment:
You're only testing one muscle, so this if-statement seems unnecessary.

Reply:
Missed this in my gutting of testProbes. Gone!